src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/filtering/MinAfFilter.java

```diff
@@ -0,0 +1,49 @@
+package org.broadinstitute.hellbender.tools.walkers.mutect.filtering;
+
+import htsjdk.variant.variantcontext.VariantContext;
+import org.apache.commons.lang.mutable.MutableBoolean;
+import org.broadinstitute.hellbender.utils.GATKProtectedVariantContextUtils;
+import org.broadinstitute.hellbender.utils.variant.GATKVCFConstants;
+
+import java.util.*;
+
+public class MinAfFilter extends HardFilter {
+    private final double minAf;
+
+    public MinAfFilter(final double minAf) {
+        this.minAf = minAf;
+    }
+
+    @Override
+    public ErrorType errorType() { return ErrorType.ARTIFACT; }
```

Comment:
It's your judgment call, but I'm torn between `ErrorType.ARTIFACT` and `ErrorType.SEQUENCING`.

Reply:
Since these are mostly mapping artifacts (from NuMTs) I was leaning towards `ARTIFACT`,  but maybe let me know if you still think this should be `SEQUENCING`.

Reply:
Oh I see your comment below, should these be `NON_SOMATIC`? Really low VAF things includes sequencing errors, mapping errors, etc.